### ISSUE TITLE
Different Library structure in `cadet` and `cadet-frontend`

### ISSUE THREAD

**Issue by indocomsoft** [source-academy/cadet#43]
According to https://gist.github.com/remo5000/cd6923c6fa004442bbc7c633718966fc, there is a difference between the library structure in `cadet` and `cadet-frontend`. 

Since you are the ones using the library, I'll change the version stored in the backend to your version. Sounds good? @ningyuansg @remo5000

**Comment by ningyuansg** [comment]
We made a typo there, there's an additional field `sourceChapter`.

It should be

```
type Library = {
  chapter: number
  externals: string[]
  files: string[]
  globals: string[]
}
```

So the change would just be renaming week to chapter. Sound good.

**Comment by remo5000** [comment]
Updated typo. The rest looks good. In terms of the backend, you'll be keeping the same Assessments context right?

**Comment by indocomsoft** [comment]
@remo5000 
Right now what used to be called `Assessment` is called `Mission` which can be either a mission/sidequest/path/contest (I know, it's kind of a misnomer from Evan)

But I guess it might be better if we just rename the `Mission` model back to `Assessment`.

But yeah it's just gonna be one "`Mission`" controller for all 4 kinds of assessments

**Comment by ningyuansg** [comment]
+1 on renaming the context 'mission' to 'assessment'

**Comment by indocomsoft** [comment]
Fixed by 72a8a1e
